source/includes/agents/_60-bulk-delete.md.erb

```diff
@@ -0,0 +1,47 @@
+## Bulk Delete agents
+
+```shell
+curl 'https://ci.example.com/go/api/agents' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v2+json' \
+      -H 'Content-Type: application/json' \
+      -X DELETE \
+      -d '{
+            "uuids": [
+              "adb9540a-b954-4571-9d9b-2f330739d4da",
+              "adb528b2-b954-1234-9d9b-b27ag4h568j1"
+            ]
+          }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v2+json; charset=utf-8
+```
+
+```json
+{
+  "message": "Deleted 2 agent(s)."
+}
+```
+
+Deletes a group of agents.
+
+<aside class="notice">
+  You must first <a href='#bulk-update-agents'>disable the agents</a> and ensure that their
+  <a href='#get-all-agents'>status</a> is not <code>Building</code>, before attempting to deleting them.
```

Comment:
`#the-agent-object` will give the status of only one agent, whereas we need the status of all the agents that needs to be deleted so i used `#get-all-agents`